docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -950,3 +950,29 @@ by the presence of the marker file `META-INF/panache-archive.marker`. Panache in
 annotation processor that will automatically create this file in archives that depend on
 Panache (even indirectly). If you have disabled annotation processors you may need to create
 this file manually in some cases.
+
+WARNING: If you include the jpa-modelgen annotation processor this will exclude the Panache
+annotation processor by default. If you do this you should either create the marker file
+yourself, or add the `quarkus-panache-common` as well, as shown below:
```

Comment:
s/or add the `quarkus-panache-common` as well/or add the `quarkus-panache-common` APT processor as well/